test/Makefile

```diff
@@ -1,24 +0,0 @@
-CXX ?= c++
```

Comment:
The tests mentioned in this Makefile and the Makefile itself are not obsolete, they were all both added recently by me. They also run in CI.

The other test files in this directory can be deleted.

Reply:
Thanks for the heads up, adjusted